modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/paywallreviews/UpgradeLayoutReviews.kt

```diff
@@ -53,6 +61,17 @@ fun UpgradeLayoutReviews(
     canUpgrade: Boolean,
     modifier: Modifier = Modifier,
 ) {
+    val context = LocalContext.current
+
+    val reviews: List<ReviewData> = listOf(
+        review1,
+        review2,
+        review3,
+        review4,
+        review5,
+        review6,
+    )
```

Comment:
Is this list going to be dynamic or hard-coded? If it is only static I think it would be worth moving it out the composable function so it is not recreated with every recomposition.